1.9/administration/haproxy-adminrouter.md

```diff
@@ -98,6 +98,7 @@ These instructions provide a tested [HAProxy](http://www.haproxy.org/) configura
       # Option 2: use TLS-encrypted communication with DC/OS Admin Router, but do
       # not perform server certificate verification (warning: this is insecure, and
       # we hope that you know what you are doing).
+      # server dcos-1 1.2.3.4:443 ssl verify none
       # Rewrite response Location header if it contains an absolute URL
```

Comment:
Let's add this newline again before
```
# Rewrite ...
```
for visually separating independent things.